index.html

```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
        <!-- <li><a href="" class="link"><i class="material-icons">feedback</i></a> </li> -->
         </ul>
          </div>
+</section>
 
 
-</section>   
+<section class="works">
+    <div class="project1">
```

Comment:
Kindly check and add the background color of `#E5E5E5` between the two sections and in-between each the project cards. You can see the difference in the images shown below

Your design
![Screenshot from 2022-09-23 19-02-17](https://user-images.githubusercontent.com/65233860/192039083-cae71a33-11ea-4446-8dc9-a128c061879f.png)


Figma design
![Screenshot from 2022-09-23 19-01-40](https://user-images.githubusercontent.com/65233860/192039116-c6c5e3ae-247c-4926-81fb-35a7d5819799.png)